AdvancedSettings - Cleanup not used values

fritsch opened the pull request:
Remove some video / audio settings that are not in use anymore.

fritsch committed:
AdvancedSettings: Remove force vaapi - not used anymore

fritsch committed:
AdvancedSettings: Remove FfmpegVDA setting

fritsch committed:
AdvancedSettings: Cleanup audio settings not in use anymore

fritsch commented:
Anything I have overlooked? Good to go?

Rechi commented:
`m_videoPPFFmpegDeint`, `m_DXVAForceProcessorRenderer`, `m_mediacodecForceSoftwareRendering`, `m_controllerDeadzone`, `m_bVideoLibraryExportAutoThumbs`, `m_curlDisableHTTP2`, `m_RestrictCapsMask`, `m_sleepBeforeFlip`, `m_bAllowDeferredRendering`, `m_screenAlign_xOffset`, `m_screenAlign_yOffset`, `m_screenAlign_xStretchFactor`, `m_screenAlign_yStretchFactor`, `m_databaseSavestates` and `m_useDisplayControlHWStereo` are unused too.

fritsch commented:
Will add them. Thanks.

fritsch committed:
AdvancedSettings: Further cleanup of unused variables

Remove:
m_videoPPFFmpegDeint, m_DXVAForceProcessorRenderer, m_mediacodecForceSoftwareRendering,
m_controllerDeadzone, m_bVideoLibraryExportAutoThumbs, m_RestrictCapsMask, m_sleepBeforeFlip,
m_bAllowDeferredRendering, m_screenAlign_xOffset, m_screenAlign_yOffset, m_screenAlign_xStretchFactor,
m_screenAlign_yStretchFactor, m_databaseSavestates and m_useDisplayControlHWStereo
